dvc/command/repro.py

```diff
@@ -48,6 +48,7 @@ def run(self):
                 if self.args.metrics:
                     metrics = self.repo.metrics.show()
                     show_metrics(metrics)
+                logger.info("Don't forget to execute 'dvc push'")
```

Comment:
Feels more natural to use `run` here.
```suggestion
                logger.info("Don't forget to run 'dvc push'.")
```